app/src/js/directives/dropletPreviewZip.js

```diff
@@ -0,0 +1,17 @@
+angular.module('media_manager')
+.directive('dropletPreviewZip', [function(){
+  var zipImage = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAGAAAABgCAYAAADimHc4AAADaUlEQVR42u2br48aQRiGVyAQJxAIRMWJE0gEAkmKOHECcWkQCARNKtHNCcQJBAJxruaSGv6AihOXhiYnLylpmv5IEKRBIHrpNeES2tJmOx8XyLJlj51ll212nzd5E8jszma/Z2e+mcmMYRjGJ2UTh2KJ/fwHCkcmAAAAAAAAAAAIAABAAAAAAgAAEAAAgAAAAAQAACAAAAABAAAIAABAAAAAAgAA0H8CoKD8Uvmz8tSI9yZbef+x8ivlx0EDSCq/IOiO/q38XvlRUABeK88I9EZ/cwFBG8ATgu/aP5TfKif8BPCBwGr5i/IzPwH8IahaHil3/QRAUPV9EyqA/f39rby3t7dSXyqVWpbJb/vzEomE67rT6fSuIIQHYFvVarWV+prN5rJMfq8DrqPZbGb2+33z9PTUzGQyANg1AKsmk4lZLpejBeD8/FzL8jVadXR05BmABHTT866uruatwNoicrlcdADoOJvNzoO2kATIfo0OgOFw6Oq5EvDxeLy87/LyMn4ApP8djUbLIFxfX5vJZHInAMSlUmml5Uk9sQEgIxfpChaSr9EpIQYFQCzXL3R8fBwfAGdnZyt9cKFQcLw2SAC9Xs8x+UcWgLyoVfV6/cHrgwRgHQDEogXk83lzOp0uX1pawqZ7ggJwcHCw8iHI/0gDsCddaf6SC8IAIDNqaw6SAUCkR0ESaGt/KyDczkJ1AEgyLxaLjq5UKma73V4ZgoqkLNIArElXxv06E58gZ8KiRqMRrZnwpqRbrVa17g8KgHRBh4eH0VsLeijptlot7Tp0ANze3s6vcbJ8DBL0HayKhg/AnnQvLi5cJd1dDkMjC8CedAeDwdq1fAAEBMCedGXRzWtdANgy6dqXlwEQIAB70j05Odn6ZQDgMemuW9sHQIAA7F2PwJCA6Lrb7QLADwBeJaMnAHgAIBMcCd627nQ6/4BdlK1bs5eub1Fubz2xHYbG3Hd+ArgjoNpb1ft+AugTVO29oW0/AVSVfxJY1/5uPHxGwNMBjTfsknZ9PqDsIqdqA0grv1P+RZDXWuLyVbnoclDj6ZCenPp4atwf0KM13FtODn1Ufm7cn6MzggSA/BEAAAAAAAAAAAgAAEAAAAACAAAQAACAAAAABAAAIAAAAAEAAAgAAEAAAAACAACQ3wDGBhtrw/L4L9MAX9xP8r3XAAAAAElFTkSuQmCC';
```

Comment:
maybe drop a comment on where this came from / how to replicate it slightly differently?